### PR TITLE
Forbid now includes ReturnUrl

### PR DESCRIPTION
**Pull request by HaoK**
cc @Tratcher

### CODE REVIEW

File: src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs [hunk 1]
```diff
@@ -327,14 +327,9 @@ private static bool IsHostRelative(string path)
 
         protected override async Task<bool> HandleForbiddenAsync(ChallengeContext context)
```
Comment (Tratcher) [inline src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs]: Shouldn't this completely mimic HandleUnauthorizedAsync? e.g. allowing an alternate redirect uri, etc..

Comment (HaoK) [inline src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs]: Sure why not, updated

**Comment by Tratcher** [comment]
:shipit:

**Comment by HaoK** [comment]
dc6e916bd465628c3e6441925cdb16239a0b82b9

**Pull request closed without merging**